go/v1beta1/main/init.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2017 The Grafeas Authors. All rights reserved.
+//
```

Comment:
Can we just combine init.go with main.go, it makes it easier to see what's happening in main.

Reply:
good idea